src/xma/include/lib/xma_utils.hpp

```diff
@@ -22,15 +22,7 @@
 #include <map>
 
 namespace xma_core {
-   static const std::map<XmaSessionType, const std::string> sessionMap = {
-      { XmaSessionType::XMA_SCALER, "scaler"},
-      { XmaSessionType::XMA_ENCODER, "encoder"},
-      { XmaSessionType::XMA_DECODER, "decoder"},
-      { XmaSessionType::XMA_FILTER, "filter"},
-      { XmaSessionType::XMA_KERNEL, "kernel"},
-      { XmaSessionType::XMA_ADMIN, "admin"},
-      { XmaSessionType::XMA_INVALID, "invalid"}
-   };
+   extern const std::map<XmaSessionType, const std::string> sessionMap;
```

Comment:
It would be best to not refer to variables defined in other source files.  See below on how you might possible wish to address this.

Reply:
excellent suggestion. done